libs/ui/calendar/brain/src/index.ts

```diff
@@ -0,0 +1,37 @@
+export * from './lib/brn-calendar-cell-button.directive';
```

Comment:
```suggestion
export * from './lib/brn-calendar-cell.directive';
export * from './lib/brn-calendar-cell-button.directive';
```
We need to export BrnCalendarCellDirective here to use it in the HlmCalendarComponent